### ISSUE TITLE
Support for additional account types

### ISSUE THREAD

**Issue by bdowling** [alpacahq/Alpaca-API#57]
It would be great to offer additional account types, such as..

- Joint 
- IRAs,
- Roth IRAs
- Custodial Roth IRAs
- Cash

**Comment by bdowling** [comment]
#24 related

**Comment by JonFurmanski** [comment]
+1 for this request

**Comment by SierraVista90** [comment]
Yes, +1 for this request.   My family has many retirement accounts would like to use Alpaca for investment.   Please add this feature.

**Comment by corakwue** [comment]
+1  to this

**Comment by Germs2004** [comment]
I like this request too.  If you enable cash accounts, please configure them to disable the PDT restrictions related to transactions per day and the $25K deposit requirement.

**Comment by cegfault** [comment]
+1 to this.
Cash accounts without PDT restrictions are huge for me.
Anything Roth would be nice :)
